sphinx/source/docs/user_guide/examples/extensions_example_ticking.py

```diff
@@ -1,29 +1,30 @@
 from bokeh.io import show
 from bokeh.models import TickFormatter
 from bokeh.plotting import figure
+from bokeh.util.compiler import TypeScript
 
-JS_CODE = """
-import {TickFormatter} from "models/formatters/tick_formatter"
+TS_CODE = """
+import { TickFormatter } from "models/formatters/tick_formatter"
```

Comment:
There are a few whitespace inconsistencies. We don't enforce such yet, but it would be good to keep things consistent. So, no whitespace between `{}` in imports. One space before block opening `{` and after `:` (type declaration, literal object field, etc.).